3rdParty/libpng/CMakeLists.txt

```
@@ -19,10 +19,14 @@ set(PNG_TESTS OFF)
 set(PNG_EXECUTABLES OFF)
 set(SKIP_INSTALL_PROGRAMS ON)
 
+if(IOS AND CMAKE_OSX_ARCHITECTURES STREQUAL "arm64")
+  set(PNG_ARM_NEON "on")

```

Normal variables do not override cache variables until CMake 3.21 (CMP0126).
To make this compatible with older CMake versions, do:
```suggestion
  set(PNG_ARM_NEON "on" CACHE STRING "" FORCE)
```